integration-tests/endToEndFlow.spec.ts

```diff
@@ -69,7 +70,7 @@ describe('Query results generated', () => {
     expect(rows[0].name).toBeDefined()
   })
 
-  it('Query does not match data - Empty CSV file should be downloaded', async () => {
+  it.skip('Query does not match data - Empty CSV file should be downloaded', async () => {
```

Comment:
Since this isn't running in a pipeline, you probably don't need to add any skips to the code for now. You can use Jest's various CLI flags to achieve the same results, and there's less risk of us forgetting to remove later.

Reply:
I forgot to remove that! Thanks.